<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mw_gestaoEstoque</class>
 <widget class="QDialog" name="mw_gestaoEstoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Produto</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>691</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_novoproduto">
    <attribute name="title">
     <string>Novo Produto</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>20</y>
       <width>651</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="codProduto">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Código do Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_codigoproduto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="descricaoProduto">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Descrição Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_descricaoproduto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="qtdeEstoque">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Qtde Estoque</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_qtdeestoque"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="valorCompra">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Valor de Compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_valorcompra"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="valorVenda">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Valor de Venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_valorvenda"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Fornecedor">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_fornecedor"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_novoproduto">
          <property name="text">
           <string>Novo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_gravarnovoproduto">
          <property name="text">
           <string>Gravar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gestaoprodutos">
    <attribute name="title">
     <string>Gestão Produtos</string>
    </attribute>
    <widget class="QTableWidget" name="tw_ge_produtos">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>121</y>
       <width>301</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>371</x>
       <y>24</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Filtrar por:</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>112</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rb_ge_codigo">
         <property name="text">
          <string>Código Produto</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_ge_produto">
         <property name="text">
          <string>Descrição Produto</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLineEdit" name="txt_ge_filtrar">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>73</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>534</x>
       <y>46</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Filtrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ge_filtrar">
     <property name="geometry">
      <rect>
       <x>595</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Filtrar</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>331</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Código do Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ge_codigoproduto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ge_produto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Quantidade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ge_qtde"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Valor Compra</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ge_valorcompra"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Valor Venda</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ge_valorvenda"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Fornecedor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ge_fornecedor"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_ge_gravar">
          <property name="text">
           <string>Gravar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ge_excluir">
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
